rust/pragma-utils/src/conversions/alloy.rs

```diff
@@ -0,0 +1,7 @@
+use alloy::primitives::U256;
+use anyhow::Context;
+
+pub fn hex_str_to_u256(s: &str) -> anyhow::Result<U256> {
```

Comment:
Add rustdoc

Maybe rename to hex_str_to_alloy_u256

Reply:
should be good since it's the `alloy` module already? but added rustdoc